content/en/docs/faq/devops/create-devops-kubeconfig-on-aws.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Create a DevOps Kubeconfig on AWS"
+keywords: "KubeSphere, Kubernetes, DevOps, Kubeconfig, AWS"
+description: "How to create a DevOps kubeconfig on AWS"
+linkTitle: "Create a DevOps Kubeconfig on AWS"
+Weight: 16820
+---
+
+This tutorial demonstrates how to create a DevOps kubeconfig on AWS.
+
+## Prerequisites
+
+- You have an AWS cluster with KubeSphere installed. For more information about how to install KubeSphere on AWS, refer to [Deploy KubeSphere on AWS EKS](../../../installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-eks/).
+- You have enabled [the KubeSphere DevOps system](../../../pluggable-components/devops/).
+- You have created a project. This tutorial uses the project `kubesphere-sample-dev` as an example.
```

Comment:
Create a project here or DevOps project? I notice in line 80 you wrote `Log in to your KubeSphere console and go to your DevOps project.`

Reply:
The project here is different from a DevOps project. I think the creation of `devops-deploy.yaml` file and the following `kubectl apply` are to enable the deployment of applications into the project `kubesphere-sample-dev` when creating a kubeconfig credential in a DevOps project. I'll add some words at the beginning.